scripts/forge-scripts/DeployTreasuryGovernor.s.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+// slither-disable-start reentrancy-benign
+
+pragma solidity 0.8.16;
+
+import {BaseGovernorDeployer} from "scripts/forge-scripts/BaseGovernorDeployer.sol";
+import {L2ArbitrumGovernorV2} from "src/L2ArbitrumGovernorV2.sol";
+
+// Concrete deployment script for the Arbitrum L2 Core Governor.
+contract DeployTreasuryGovernor is BaseGovernorDeployer {
+    function NAME() public pure override returns (string memory) {
+        return "Treasury L2ArbitrumGovernor V2";
```

Comment:
nit: both currently deployed governors show `L2ArbitrumGovernor` as the name, with a VERSION of 1. what are your thoughts of leaving the name the same, and/or bumping the version?

also, might this name be better defined in the GovernorConstants file, leaving 1 file w/ the essential args to review?